exdb/emails.py

```diff
@@ -0,0 +1,181 @@
+                experience_count = len(experiences)
+
+                status_to_display = [_('Pending Approval'), _('Needs Evaluation')]
+                experience_dict = OrderedDict()
+                for status in status_to_display:
+                    experience_dict[status] = []
+                for experience in experiences:
+                    if experience.needs_evaluation():
+                        experience_dict[_('Needs Evaluation')].append(experience)
+                    else:
+                        experience_dict[_('Pending Approval')].append(experience)
+
+                html = render_to_string('exdb/emails/daily.html',
+                                        {'experience_dict': experience_dict,
+                                         'experience_count': experience_count,
+                                         'url_prefix': settings.URL_PREFIX})
+                text = strip_tags(html)
+                recipients = (user.email,)
+                emails.append((subject, text, html, from_email, recipients))
+            send_mass_mail(emails)
+            return len(emails)
+        return 0
+
+
+class ExperienceStatusUpdate(EmailTaskBase):
+    """
+    This email should be sent to any user who has an experience they
+    authored that has changed in status to either approved or denied.
+    """
+    task_name = "Status Update"
+
+    def get_experiences(self):
+        return Experience.objects.filter(status__in=('de', 'ad'), needs_author_email=True)
+
+    def send(self, *args, **kwargs):
+        emails = []
+        from_email = settings.SERVER_EMAIL
+        subject = settings.EMAIL_SUBJECT_PREFIX + ' Experience status updated'
+        for experience in self.get_experiences():
+
+            html = render_to_string('exdb/emails/status_change.html',
+                                    {'experience': experience,
+                                     'url_prefix': settings.URL_PREFIX})
+            text = strip_tags(html)
+            recipients = (experience.author.email,)
+            emails.append((subject, text, html, from_email, recipients))
+
+            # Reset the bool so we do not send it out again
+            experience.needs_author_email = False
+        send_mass_mail(emails)
+        return len(emails)
+
+
+class EvaluateExperience(EmailTaskBase):
+    """
+    This email should be sent as soon as an experience end time passes.
+    The email is set to the author and each of the planners for each experience.
+    Once it is sent the email will be sent every 24 hours until the experience is evaluated.
+    OR
+    The email will be sent to every author and planner of an experience if 24 hours has passed
+    since the last evaluation reminder email and they have yet to evaluate it.
+    """
+    task_name = "Evaluation Email"
+
+    def get_experiences(self):
+        one_day = (now() - timedelta(days=1))
+        return Experience.objects.filter(
+            Q(status='ad',
```

Comment:
This could be DRY'd out some:
```python
.filter(
     Q(last_evaluation_email_datetime__isnull=True) |
     Q(last_evaluation_email_datetime__range=((one_day - timedelta(minutes=5)), one_day)),
     status='ad', end_datetime__lt=now()
)
```